pkg/loadbalancer/experimental/frontend.go

```diff
@@ -89,6 +89,7 @@ func (fe *Frontend) TableHeader() []string {
 		"PortName",
 		"Backends",
 		"Status",
+		"StatusKind", // FIXME: Figure out a better way to do this.
```

Comment:
This was added so that the tests can just check whether the frontend is marked "Done" without having to deal with the non-deterministic timestamp in `Status.String()` ("Done (10ms ago"). I'm kind of inclined to just leave it this way even though you'll then see "Done (10ms ago)   Done" in the table output. Alternative would be to break the Status into multiple columns (StatusKind, StatusSince, StatusMessage)...

Reply:
could we just split the output by space in this case for example when checking the output? splitting the columns in StatusKind / StatusSince sounds reasonable to me tho.

Reply:
Split this into "Status", "Since" and "Error"